pkg/kv/kvclient/kvcoord/dist_sender.go

```diff
@@ -139,6 +139,24 @@ errors as 'roachpb.InternalErrType'.
 		Measurement: "Errors",
 		Unit:        metric.Unit_COUNT,
 	}
+	metaDistSenderRangefeedTotalRanges = metric.Metadata{
+		Name: "distsender.rangefeed.catchup_ranges",
+		Help: `Number of ranges executing rangefeed
+
+This counts the total number of ranges that are rangefeed
```

Comment:
Seems like we might be missing a word here.
```suggestion
This counts the total number of ranges that are rangefeed
```